docs/creating-bots/csharp/unit-testing.mdx

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_label: 'Unit Testing'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Defining Bots for Unit Testing
```

Comment:
I've haven't brought this remark up and that was my fault - what exactly are unit testing bots? I believe by definition, bots would never be involved in unit testing?

Reply:
Per our huddle, `PlayTest Bots` are now referred to as `Adaptive Bots` to describe how they adapt to the current game state, and `Validation Bots` and now referred to as `Scenario Bots` so there is no confusion about Adaptive Bots being able to also use validations, and to describe how these bots run through pre-determined scenarios.